src/index.ts

```diff
@@ -591,9 +591,9 @@ class Logging {
     }
 
     reqOpts.resourceNames = arrify(reqOpts.resourceNames!);
-    this.projectId = await this.auth.getProjectId();
-    const resourceName = 'projects/' + this.projectId;
-    if (reqOpts.resourceNames.indexOf(resourceName) === -1) {
+    if (reqOpts.resourceNames.length === 0) {
```

Comment:
I would definitely add a comment just inside this if statement explaining the reason why we only add the resourceName when the resourceNames array is empty.